analysis/hyp_reg_dataset.py

```diff
@@ -0,0 +1,56 @@
+from ehrql import codelist_from_csv
+from ehrql import create_dataset
+from ehrql.tables.tpp import (
+    clinical_events, 
+    patients,
+    practice_registrations,
+)
+
+# Hypertension codes
+hyp_cod = codelist_from_csv(
+    "codelists/nhsd-primary-care-domain-refsets-hyp_cod.csv",
+    column="code"
+)
+# Hypertension resolved codes
+hypres_cod = codelist_from_csv(
+    "codelists/nhsd-primary-care-domain-refsets-hypres_cod.csv",
+    column="code"
+)
+
+index_date = "2024-03-31"
+dataset = create_dataset()
+
+### variables ###
+
+# age of the patient in full years at the achievement date
+dataset.pat_age = patients.age_on(index_date)
```

Comment:
I dont think the age varible is needed for the hypertension register (but it will be important for the clinical rules), so we could remove it here

Reply:
but we may want to break the results down by age, in which which case we would have to include the variable in the dataset definition

Reply:
I have removed the age variable.